app/views/ems_container/_form.html.haml

```diff
@@ -84,7 +84,8 @@
                      "checkchange"                 => "",
                      "required"                    => "",
                      "selectpicker-for-select-tag" => "")
-
+    %input{:type => 'text', :id => "current_tab", :name => "current_tab", "ng-model" => "currentTab",
+           "ng-hide" => "true"}
```

Comment:
can you change this to the standard way of writing a hash -  which is 1 key/value per line?

or include everything in 1 line.